irrep/__kpoint.py

```diff
@@ -26,6 +26,9 @@
 from .__aux import bohr
 from scipy.io import FortranFile as FF
 from lazy_property import LazyProperty
+
+Ry2eV=27.2
```

Comment:
Is it in Ry or Ha ?   There is already a variable Hartree_eV, imported on line 23 from __read_files module, just use it. It has much more digits, btw.